source/modules/configuration/index.rst

```diff
@@ -8,9 +8,11 @@ Configuration module gives access to all GLPI configuration options: global conf
 
    authentication/index
    dropdowns/index
+   components
    collectors
    fields-unicity
    general/index
+   external_links
```

Comment:
Either you should rename `source/modules/configuration/external_link.rst` to `source/modules/configuration/external_links.rst` or remove trailing `s` here